src/registerServiceWorker.js

```diff
@@ -0,0 +1,125 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
Why do you decided to user SErvice Workers ?

Reply:
The same reason  - I used a react boilerplate - service worker was provided out of the box.
Its actually not used (commented) and not required here.
I have removed it.